community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/README.md

```diff
@@ -1,4 +1,39 @@
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Please fix it.

Reply:
Done, added it back after my addition as per offline chat.